### PR TITLE
added task_10

### PR DESCRIPTION
**Pull request by katy-lix**
added task_10

### CODE REVIEW

File: Task_10/index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
```
Comment (Killerkip10) [inline Task_10/index.html:1]: Page is not adaptive

File: Task_10/index.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style1.css" type="text/css"/>
+</head>
+<body>
+    <header>
+
+    <div class="conteiner_heder">
```
Comment (Killerkip10) [inline Task_10/index.html]: container_header

File: Task_10/index.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        <img src="./img/rabbit-mouth.png" class="header_rabbit" >
+     <div class="header_text">
+         <span class="orange">orange</span><br>
+         <span class="rabbit_text4" >RABBIT</span>
```
Comment (Killerkip10) [inline Task_10/index.html]: What is rabbit_text4?) Please use understandable class names.
Or you can use pseudo classes

File: Task_10/style1.css [hunk 4]
```diff
@@ -0,0 +1,551 @@
...
+    top: 8px;
+  }
+  
+  .menu__box {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    left: -100%;
+
```
Comment (Killerkip10) [inline Task_10/style1.css:185]: Please remove this empty rows

File: Task_10/index.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                <li><a class="menu__item" href="#">work</a></li>
+                <li><a class="menu__item" href="#">blog</a></li>
+        </ul>
+      </div>
+
+    <!-- \\\\\\\\\\\\\\\\\\\\\\\\\\\ -->
+
+           <div class="icons">
+                <figure>
+            <div class="img1"></div>
```
Comment (Killerkip10) [inline Task_10/index.html]: Please use understandable class names

File: Task_10/index.html [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+            <figcaption class="figure_text">home</figcaption>
```
Comment (Killerkip10) [inline Task_10/index.html:41]: Please create class for all nav items with common css prop.

File: Task_10/index.html [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+                 <figcaption class="figure_text">blog</figcaption>
+                 </figure>
+                 <figure>
+             <div class="img5"></div>
+                 <figcaption class="figure_text">contact</figcaption>
+                 </figure>  
+        </div>
+    
+</header>
+<div class="div_width">
```
Comment (Killerkip10) [inline Task_10/index.html]: div_width ?)))))

File: Task_10/style1.css [hunk 8]
```diff
@@ -0,0 +1,551 @@
...
+}
+@media ( max-width:800px){
+    .flex_text1{
+        display: flex;
+        margin-left: 50%;
+        padding-top: 0;
+        justify-content: center;
+}}
+.left1{
+    position: absolute;
```
Comment (Killerkip10) [inline Task_10/style1.css:264]: I think you can use flex in this case

File: Task_10/style1.css [hunk 9]
```diff
@@ -0,0 +1,551 @@
...
+}}
+.left1{
+    position: absolute; 
+    color:orange;
+    width: 400px;
+    background-color:rgba(29, 28, 28, 0.884);
+}
+
+.left2{
+    position: absolute;
```
Comment (Killerkip10) [inline Task_10/style1.css:272]: Also use flex

File: Task_10/style1.css [hunk 10]
```diff
@@ -0,0 +1,551 @@
...
+    color:black;
+    background-color:orange;
+    width: 400px;
+    margin-top: 100px;
+}
+
+@media ( max-width:800px){
+    .left1{
+        width:250px;
+        font-size: 95%;
```
Comment (Killerkip10) [inline Task_10/style1.css]: Pleeeease don't use `%` without the need. Use `px`

File: Task_10/style1.css [hunk 11]
```diff
@@ -0,0 +1,551 @@
...
+}
+@media (max-width: 800px){
+
+    .span_center{
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;    
+    }
+}
+/* .rabbit1.rabbit2.rabbit3 {
```
Comment (Killerkip10) [inline Task_10/style1.css]: Please remove all commented code in final homework version

File: Task_10/style1.css [hunk 12]
```diff
@@ -0,0 +1,551 @@
...
+    height: 320px;
+    cursor: pointer;
+    background-color: orange;
+    border-radius: 220px;
+    border:solid orange;
+    background-size: cover;
+    background-repeat: no-repeat;
+
+   } */
+   .rabbit1{
```
Comment (Killerkip10) [inline Task_10/style1.css:356]: All rabbits almost the same. Create one class for all rabbits

File: Task_10/index.html (hunk 1 above)
Comment (Killerkip10) [inline Task_10/index.html:1]: I mole you, align all code

File: Task_10/index.html [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+                <!-- ///////////////////////////////// -->
+                <div class="text_input1">Send us a message...</div>
+                
+
+                
+        <div class="container_input">
+            <form>
+              <div class="flex-outer">
+                  <div>
+                  <label> Name:  </label>
```
Comment (Killerkip10) [inline Task_10/index.html:120]: Label and textarea don't connect with each other

Comment (katy-lix) [reply Task_10/index.html:120]: seems to have fixed

File: Task_10/style1.css (hunk 4 above)
Comment (katy-lix) [reply Task_10/style1.css:185]: seems to have fixed

File: Task_10/style1.css (hunk 8 above)
Comment (katy-lix) [reply Task_10/style1.css:264]: It seems to me that I did not correct it correctly.

File: Task_10/style1.css (hunk 9 above)
Comment (katy-lix) [reply Task_10/style1.css:272]: It seems to me that I did not correct it correctly.

File: Task_10/style1.css (hunk 12 above)
Comment (katy-lix) [reply Task_10/style1.css:356]: I tried this way and through (* and ^), but it did not work. I'm sorry

File: Task_10/index.html (hunk 6 above)
Comment (katy-lix) [reply Task_10/index.html:41]: seems to have fixed

File: Task_10/index.html (hunk 1 above)
Comment (katy-lix) [reply Task_10/index.html:1]: seems to have fixed

File: Task_10/index.html (hunk 7 above)
Comment (katy-lix) [reply Task_10/index.html]: fixed

File: Task_10/style1.css (hunk 10 above)
Comment (katy-lix) [reply Task_10/style1.css]: fixed

File: Task_10/index.html (hunk 1 above)
Comment (katy-lix) [reply Task_10/index.html:1]: seems to have fixed

Comment (Killerkip10) [reply Task_10/index.html:1]: nearly

File: Task_10/index.html (hunk 3 above)
Comment (Killerkip10) [reply Task_10/index.html]: not fixed

**Review by Killerkip10 — COMMENTED** [review]
## Task 10
- Page partially adaptive. (-5 points);
- Not understandable class names. (-5 points);
- Inputs and labels are not connected. (-5 points);
- Html validator errors. (-5 points);

### Total 80 points

**Pull request merged**